qa/tasks/cephfs/test_misc.py

```diff
@@ -325,6 +325,29 @@ def file_sync_thread(mount, path):
         self.mount_a.run_shell(["rm", "-rf", dir_path])
 
 
+    def _session_client_ls(self, cmd):
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.rank_asok(cmd)
+        mount_a_mountpoint = self.mount_a.mountpoint
+        mount_b_mountpoint = self.mount_b.mountpoint
+        self.assertIsNotNone(info)
+        for i in range(0, len(info)):
+            self.assertIn(info[i]["client_metadata"]["mount_point"], 
+                             [mount_a_mountpoint, mount_b_mountpoint])        
+        info = self.fs.rank_asok(cmd + [f"id={mount_a_client_id}"])
+        self.assertEqual(len(info), 1)
+        self.assertEqual(info[0]["id"], mount_a_client_id)
+        self.assertEqual(info[0]["client_metadata"]["mount_point"], mount_a_mountpoint)
+        info = self.fs.rank_asok(cmd + ['--cap_dump'])
+        for i in range(0, len(info)):
+            self.assertIn("caps", info[i])
+
+    def test_session_ls(self):
+        self._session_client_ls(['session', 'ls'])
+
+    def test_client_ls(self):
```

Comment:
@neesingh-rh PTAL at the failure - https://pulpito.ceph.com/yuriw-2023-08-07_23:35:53-fs-wip-yuri10-testing-2023-08-01-0753-pacific-distro-default-smithi/7362351/

```
2023-08-08T00:09:03.543 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2023-08-08T00:09:03.543 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_569a240beeab1f0e3097de970ef4b8c68c2b492a/qa/tasks/cephfs/test_misc.py", line 349, in test_client_ls
2023-08-08T00:09:03.544 INFO:tasks.cephfs_test_runner:    self._session_client_ls(['client', 'ls'])
2023-08-08T00:09:03.544 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_569a240beeab1f0e3097de970ef4b8c68c2b492a/qa/tasks/cephfs/test_misc.py", line 341, in _session_client_ls
2023-08-08T00:09:03.544 INFO:tasks.cephfs_test_runner:    info = self.fs.rank_asok(cmd + ['--cap_dump'])
2023-08-08T00:09:03.544 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_569a240beeab1f0e3097de970ef4b8c68c2b492a/qa/tasks/cephfs/filesystem.py", line 1237, in rank_asok
2023-08-08T00:09:03.544 INFO:tasks.cephfs_test_runner:    return self.json_asok(command, 'mds', info['name'], timeout=timeout)
2023-08-08T00:09:03.545 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_569a240beeab1f0e3097de970ef4b8c68c2b492a/qa/tasks/cephfs/filesystem.py", line 259, in json_asok
2023-08-08T00:09:03.545 INFO:tasks.cephfs_test_runner:    j = json.loads(response_data)
2023-08-08T00:09:03.545 INFO:tasks.cephfs_test_runner:  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
2023-08-08T00:09:03.545 INFO:tasks.cephfs_test_runner:    return _default_decoder.decode(s)
2023-08-08T00:09:03.545 INFO:tasks.cephfs_test_runner:  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
2023-08-08T00:09:03.546 INFO:tasks.cephfs_test_runner:    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
2023-08-08T00:09:03.546 INFO:tasks.cephfs_test_runner:  File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
2023-08-08T00:09:03.546 INFO:tasks.cephfs_test_runner:    raise JSONDecodeError("Expecting value", s, err.value) from None
2023-08-08T00:09:03.546 INFO:tasks.cephfs_test_runner:json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

Reply:
Another failure - https://pulpito.ceph.com/yuriw-2023-08-07_23:35:53-fs-wip-yuri10-testing-2023-08-01-0753-pacific-distro-default-smithi/7362362